README.md

```diff
@@ -6,22 +6,40 @@ Puppet module for managing Windows poweshell DSC resources.
 
 This module generates Puppet Types based on DSC resources MOF schema files.
 
-In this version, the following DSC Resources are already build and ready for usage:
-- All base DSC resources found in powershell 4. ([wmf 4.0](http://www.microsoft.com/en-us/download/details.aspx?id=40855))
-- All DSC resources found in the [DSC resource kit wave 6](http://gallery.technet.microsoft.com/DSC-Resource-Kit-All-c449312d).
+In this version, the following DSC Resources are already built and ready for use:
+- All base DSC resources found in PowerShell 5. ([WMF 5.0](https://www.microsoft.com/en-us/download/details.aspx?id=46889))
+- All DSC resources found in the [DSC Resource Kit Wave 10](https://gallery.technet.microsoft.com/scriptcenter/DSC-Resource-Kit-All-c449312d).
```

Comment:
This is not strictly true anymore given we're grabbing a copy of the latest resources from https://github.com/powershell/DSCResources